ui/apps/platform/src/Containers/Violations/ViolationsTablePage.tsx

```diff
@@ -106,12 +106,16 @@ function ViolationsTablePage(): ReactElement {
         defaultSortOption,
     });
 
+    const additionalContextFilter = getFilteredWorkflowViewSearchFilter(filteredWorkflowView);
+
     const onSearch = (payload: OnSearchPayload) => {
         onURLSearch(searchFilter, setSearchFilter, payload);
     };
 
     const onChangeFilteredWorkflowView = (value) => {
         setFilteredWorkflowView(value);
```

Comment:
Aha - one small thing related to my comment above. In this case, `value` is implicitly typed as `any`, so TypeScript is letting this code circumvent the additional type safety you had added in the new hook.